dpgen/generator/run.py

```diff
@@ -1086,6 +1101,16 @@ def _make_model_devi_native_gromacs(iter_index, jdata, mdata, conf_systems):
     else:
         model_devi_dt = jdata['model_devi_dt']
     nsteps = cur_job.get("nsteps", None)
+    lambdas = cur_job.get("lambdas", [])
+    temps = cur_job.get("temps", [])
```

Comment:
You can put the default value here... like
```suggestion
    lambdas = cur_job.get("lambdas", [1.])
    temps = cur_job.get("temps", [298.])
```
and remove lines 1106-1107, 1112-1113

Reply:
Got it.